lib/libzrepl/mgmt_conn.c

```diff
@@ -451,32 +451,22 @@ uzfs_zvol_get_ip(char *host, size_t host_len)
 	return (rc);
 }
 
-/*
- * This function suppose to lookup into zvol list to find if LUN presented for
- * identification is available/online or not. This function also need to send
- * back IP address of replica along with port so that ISTGT controller can open
- * a connection for IOs.
- */
 static int
-uzfs_zvol_mgmt_do_handshake(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
-    const char *name, zvol_info_t *zinfo)
+uzfs_zvol_mgmt_get_handshake_info(zvol_io_hdr_t *hdrp, const char *name,
+    zvol_info_t *zinfo, zvol_io_hdr_t *hdr, mgmt_ack_t *mgmt_ack)
```

Comment:
two variables of same type hdrp and hdr. They may confuse in future. Better if we rename the one which already has payload.